app/uk/gov/hmrc/cgtpropertydisposalsstubs/controllers/ReturnAndPaymentProfiles.scala

```diff
@@ -77,70 +78,32 @@ object ReturnAndPaymentProfiles {
       val chargeReference = "XCRG1111111112"
       ReturnProfile(
         ReturnSummary(
-          "000000000002",
-          LocalDate.of(2020, 6, 1),
-          LocalDate.of(2020, 5, 25),
-          None,
-          "2020",
-          DesAddressDetails("Acme Ltd", Some("1 Similar Place"), Some("Southampton"), None, Some("S12 1AX"), "GB"),
-          BigDecimal("23555"),
-          Some(
+          submissionId = "000000000002",
+          submissionDate = LocalDate.of(2022, 6, 1),
+          completionDate = LocalDate.of(2022, 5, 25),
+          lastUpdatedDate = None,
+          taxYear = "2022",
+          propertyAddress = DesAddressDetails("Acme Ltd", Some("1 Similar Place"), Some("Southampton"), None, Some("S12 1AX"), "GB"),
+          totalCGTLiability = BigDecimal("23555"),
+          charges = Some(
             List(
-              Charge("CGT PPD Return UK Resident", LocalDate.of(2020, 6, 24), chargeReference)
-            )
-          )
-        ),
-        List(
-          FinancialTransaction(
-            chargeReference,
-            BigDecimal("23555"),
-            BigDecimal("23555"),
-            Some(
-              List(
-                DesFinancialTransactionItem(
-                  BigDecimal("23555"),
-                  None,
-                  None,
-                  None,
-                  Some(LocalDate.of(2020, 6, 24))
-                )
-              )
-            )
-          )
-        )
-      )
-    }
-
-    val return21 = {
-      val chargeReference = "XCRG1212121212"
-      ReturnProfile(
-        ReturnSummary(
-          "000000001212",
-          LocalDate.of(2021, 4, 10),
-          LocalDate.of(2021, 4, 5),
-          None,
-          "2020",
-          DesAddressDetails("Acme Ltd", Some("1 Similar Place"), Some("Southampton"), None, Some("S12 1AX"), "GB"),
-          BigDecimal("55555"),
-          Some(
-            List(
-              Charge("CGT PPD Return UK Resident", LocalDate.of(2021, 4, 12), chargeReference)
```

Comment:
Why is this chunk missing?

Reply:
because it's not needed and was a duplicate address